modules/custom/az_finder/js/active-filter-indicator.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file
+ * active-filter-indicator.js
+ *
+ * This file contains the JavaScript needed to display the number of active
+ * filters in the accordion headings of the AZ Finder based on configured
+ * levels.
+ */
+((Drupal, drupalSettings) => {
+  Drupal.behaviors.azFinderAccordionIndicators = {
+    attach(context) {
+      // Get the configured levels from drupalSettings.
+      const levels = drupalSettings?.azFinder?.activeFilterIndicatorLevels;
+
+      // If levels is not set, exit early.
+      if (levels === undefined || levels === null) {
+        return;
+      }
+
+      // Determine which levels to target based on configuration.
+      // levels = 0: all levels (.level-0, .level-1, .level-2, etc.)
+      // levels = 1: only level-0 (first level)
+      // levels = 2: level-0 and level-1 (first 2 levels)
+      // levels = 3: level-0, level-1, and level-2 (first 3 levels)
+      // etc.
+      let selector;
+      if (levels === 0) {
+        // Target all levels.
+        selector = 'a.collapser[class*="level-"]';
+      } else {
+        // Build selector for specific levels (0 through levels - 1).
+        const levelSelectors = [];
+        for (let i = 0; i < levels; i++) {
+          levelSelectors.push(`a.collapser.level-${i}`);
+        }
+        selector = levelSelectors.join(', ');
+      }
+
+      const toggles = once('az-filter-indicator', selector, context);
+
+      toggles.forEach((toggle) => {
+        const h3 = toggle.querySelector('h3, h4, h5, h6');
+        const { collapseId } = toggle.dataset;
+        const checkboxList = context.querySelector(`#${collapseId}`);
+
+        if (!h3 || !checkboxList) return;
+
+        const updateIndicator = () => {
+          // Count all checked checkboxes in this section and all nested sections
+          const inputs = checkboxList.querySelectorAll(
+            'input[type="checkbox"]',
+          );
+          const count = Array.from(inputs).filter(
+            (input) => input.checked,
+          ).length;
+
+          let badge = h3.querySelector('.js-az-finder-indicator');
+          if (count > 0) {
+            if (!badge) {
+              badge = document.createElement('span');
+              badge.className =
+                'js-az-finder-indicator badge bg-azurite mx-2 lh-az-normal';
```

Comment:
```suggestion
                'js-az-finder-indicator badge text-bg-azurite mx-2 lh-az-normal';
```
I think this is functionally equivalent. But it might follow our published docs for badges a little better: https://digital.arizona.edu/arizona-bootstrap/v5/docs/5.0/components/badge/#background-colors